src/main/java/seedu/address/logic/parser/UnmarkCommandParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.MarkCommand;
+import seedu.address.logic.commands.UnmarkCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new MarkCommand object
+ */
+public class UnmarkCommandParser implements Parser<UnmarkCommand> {
```

Comment:
Should we also create unit tests for `UnmarkCommandParser`?

Reply:
Same as above!